src/App.js

```diff
@@ -1,33 +1,28 @@
 import RestaurantPage from './RestaurantPage';
 import Nav from "./Nav";
 import "./App.scss"
-
+import Hero from "./Hero";
```

Comment:
Doesn't look like this is being used in this component, think it can be removed?